azure-pipelines.yml

```diff
@@ -8,20 +8,45 @@
 #- dev
 
 pool:
-    vmImage: 'vs2017-win2016'
+    name: '1ES-Hosted-AzFunc'
+    vmImage: 'MMS2019TLS'
 
 variables:
   Configuration: Release
   buildNumber: $[ counter('build', 400) ] # Start higher than our AppVeyor versions. Every build (pr or branch) will increment.
 
 steps:
+- pwsh: |
+    $releaseBranches = @('v4.x/ps7.2', 'v4.x/ps7.0', 'v3.x/ps7', 'v3.x/ps6', 'v2.x')
+    $isReleaseBuild = ($releaseBranches -contains $env:BuildSourceBranchName)
+    Write-Host "##vso[task.setvariable variable=IsReleaseBuild]$isReleaseBuild"
+    Write-Host "IsReleaseBuild: $isReleaseBuild"
+  displayName: 'Set IsReleaseBuild variable'
+  env:
+    BuildSourceBranchName: $(Build.SourceBranchName)
+
+- pwsh: |
+    Invoke-WebRequest 'https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1' -OutFile 'dotnet-install.ps1'
```

Comment:
Should we hardcode the commit of this binary? Otherwise, we risk changing it changing, and breaking, silently